README.md

```diff
@@ -169,3 +171,93 @@ things such as access keys. The test itself should error early and tell
 you what to set, so it is not documented here.
 
 For more information on Vault Enterprise features, visit the [Vault Enterprise site](https://www.hashicorp.com/products/vault/?utm_source=github&utm_medium=referral&utm_campaign=github-vault-enterprise).
+
+### Docker-based Tests
+
+We have created an experimental new testing mechanism inspired by NewTestCluster.
```

Comment:
Is this still "experimental"? We're beginning to build test scenarios on this that are not easy to build any other way and I'd consider those to be first class 🤔 . I guess it's "experimental" in the sense of "no external API guarantees" though?

Reply:
Yeah, that was my only reason for using that word.